docs/users_guide/ocean/test_groups/baroclinic_channel.md

```diff
@@ -0,0 +1,175 @@
+use_distances = False
+
+# Viscosity values to test for rpe test case
+viscosities = 1, 5, 10, 20, 200
+
+# Temperature of the surface in the northern half of the domain.
+surface_temperature = 13.1
+
+# Temperature of the bottom in the northern half of the domain.
+bottom_temperature = 10.1
+
+# Difference in the temperature field between the northern and southern halves
+# of the domain.
+temperature_difference = 1.2
+
+# Fraction of domain in Y direction the temperature gradient should be linear
+# over.
+gradient_width_frac = 0.08
+
+# Width of the temperature gradient around the center sin wave. Default value
+# is relative to a 500km domain in Y.
+gradient_width_dist = 40e3
+
+# Salinity of the water in the entire domain.
+salinity = 35.0
+
+# Coriolis parameter for entire domain.
+coriolis_parameter = -1.2e-4
+```
+
+The `nx`, `ny` and `dc` config options are set in the code.  While you can
+override these values with user config options, this isn't recommended. The
+`goal_cells_per_core` and `max_cells_per_core` are used to control how many
+MPI tasks the `rpe_test` simulations use (all other tests use hard-coded
+resources). By default, `goal_cells_per_core = 200`, indicating that the
+ocean domain will be decomposed across enough cores so there are about 200 
+cells per core.  If there aren't enough cores for this goal decomposition,
+the model will run on fewer cores but it will not allow more than
+`max_cells_per_core` cells on each core.
+
+The config options `dt_per_km` and `btr_dt_per_km` are used to determine a
+time steps that is consistent with a given resolution.  Changing these config
+options is likely to break the `restart_test`, since it assumes a time step of
+5 minutes at 10 km, consistent with the default `dt_per_km`.  A user can
+safely modify `dt_per_km` and `btr_dt_per_km` to control the time step for any
+other baroclinic channel tests.
+
+All units are mks, with temperature in degrees Celsius and salinity in PSU.
+
+## default
+
+`ocean/baroclinic_channel/10km/default` is the default version of the
+baroclinic eddies test case for a short (15 min) test run and validation of
+prognostic variables for regression testing.  Currently, only 10-km horizontal
+resolution is supported.
+
+## decomp_test
+
+`ocean/baroclinic_channel/10km/decomp_test` runs a short (15 min) integration
+of the model forward in time on 4 (`4proc` step) and then on 8 processors
+(`8proc` step) to make sure the resulting prognostic variables are
+bit-for-bit identical between the two runs. Currently, only 10-km horizontal
+resolution is supported.
+
+## thread_test
+
+`ocean/baroclinic_channel/10km/thread_test` runs a short (15 min) integration
+of the model forward in time on 1 threads per processor (`1thread` step) and
+then on 2 threads (`2thread` step) to make sure the resulting prognostic
+variables are bit-for-bit identical between the two runs. Currently, only 10-km
+horizontal resolution is supported.
+
```

Comment:
```suggestion

Note: The `ocean/baroclinic_channel/10km/thread_test/1thread` step is identical 
to `ocean/baroclinic_channel/10km/default/forward`. If
`ocean/baroclinic_channel/10km/thread_test` is included in a test suite, 
`ocean/baroclinic_channel/10km/default` may be removed from that suite.

```

Reply:
Since a user doesn't have control over what tests are in a test suite, I think we should just remove the `default` test case from the test suite.  This might be appropriate to have in the developer's guide instead.

Reply:
Thanks! Yes, it's a better fit for developer's guide.